docs/assets/javascript/elements/uxdot-pattern.css

```diff
@@ -1,89 +1,96 @@
+@import url('/styles/rh-code-block-lightdom.css');
+
 :host {
+  container: host / inline-size;
   display: block;
-  container-name: host;
-  container-type: inline-size;
-  margin-block-end: var(--rh-space-2xl, 32px);
+  margin-block-start: var(--rh-space-2xl);
 }
 
 #container {
   display: grid;
-  grid-template-columns: 1fr;
-  grid-template-areas: 'controls' 'example' 'code';
-  gap: var(--rh-space-2xl, 32px) var(--rh-space-4xl, 64px);
-  padding: var(--rh-space-2xl, 32px);
-  border: var(--rh-border-width-sm, 1px) solid var(--rh-color-border-subtle);
-  border-radius: var(--rh-border-radius-default, 3px);
-  background-color: var(--rh-color-surface);
-  color: var(--rh-color-text-primary);
-  border-color: var(--rh-color-border-subtle);
+  grid-template-columns: auto auto;
```

Comment:
we only have single column at this point.  I believe this should be the following: 
```Suggestion
grid-template-columns: auto;
```